.github/workflows/large-monorepo.yml

```diff
@@ -1,6 +1,11 @@
 name: Large Repo Benchmark
 
-on: workflow_dispatch
+on:
+  workflow_dispatch:
```

Comment:
what does this line do? not seeing much in the docs and how/if it relates to `workflow_run` event https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs

Reply:
It enables manual runs. It shows up as a dropdown option for the action on the github ui